modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer-fragment.glsl1.js

```diff
@@ -1,6 +1,13 @@
 export default `
 #define SHADER_NAME simple-mesh-layer-fs
 
+// Note(Tarek): headless-gl supports derivatives, but doesn't report it via getExtension. Awesome!
```

Comment:
Interesting. How does the `DERIVATES` macro get set then, is it just a default `define` set by the layer?

Maybe we need a polyfill for getExtension for headless-gl... Just kidding ;)